concepts/05 UI Components/DropDownButton/03 Overview/10 Key Features.md

```diff
@@ -0,0 +1,29 @@
+- **Labeling**    
+You can label the DevExtreme DropDownButton with [text](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#text) and [icons](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#icon). The component can also display a [dropdown arrow](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#showArrowIcon) to differentiate DropDownButton from the [Button](/Documentation/Guide/UI_Components/Button/Overview/) component.
+
+- **Action Binding**    
+You can specify unique DropDownButton actions for the following click handlers:
+    - [onButtonClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onButtonClick)    
+    Executed when users click the DropDownButton.
+    - [onItemClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onItemClick)    
+    Executed when users click any DropDownButton item.
+    - **items**.[onClick](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/items/#onClick)    
+    Executed when users click a specific DropDownButton item.
+
+- **Split Button Support**    
+Enable [splitButton](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#splitButton) to separate the dropdown and action buttons of the DropDownButton component.
+
+- **Selection**    
+DevExtreme DropDownButton supports selection. If [useSelectMode](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#useSelectMode) is enabled, the component displays the text and icon of the selected item. DropDownButton includes the following selection properties:
+    - [selectedItem](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#selectedItem)
+    - [selectedItemKey](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#selectedItemKey)
+    - [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#onSelectionChanged)
+
+- **Customization**    
+You can customize the appearance of DevExtreme DropDownButton elements as your needs dictate. Specify the following properties to implement your customizations:
```

Comment:
Let's leave here only templates but please add a short description what each template is for.